### YamlMime:UniversalReference
ms.openlocfilehash: 137876643e51b1b05124e0ad1af2b6bd0c26eef4
ms.sourcegitcommit: 286f52ea38c9eff2ec9d4f8cabeb86f62fd9c406
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 08/22/2018
ms.locfileid: "41687678"
items:
- uid: azure-iot-common.TwinDetachedError
  name: TwinDetachedError
  fullName: TwinDetachedError
  children:
  - azure-iot-common.TwinDetachedError.Error
  - azure-iot-common.TwinDetachedError.innerError
  - azure-iot-common.TwinDetachedError.constructor
  langs:
  - typeScript
  type: class
  summary: <span data-ttu-id="c57a5-101">전송 수준에서 작업, 시작을 다시 시도 쌍 기능이 중지 될 때 발생 하는 데 오류가 발생 했습니다.</span><span class="sxs-lookup"><span data-stu-id="c57a5-101">Error thrown when the twin feature stopped working at the transport level, requiring the client to retry starting it.</span></span>
  extends:
    name: Error
  package: azure-iot-common
- uid: azure-iot-common.TwinDetachedError.Error
  name: Error
  fullName: Error
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'static Error: ErrorConstructor'
    return:
      type:
      - ErrorConstructor
  package: azure-iot-common
- uid: azure-iot-common.TwinDetachedError.innerError
  name: innerError
  fullName: innerError
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'innerError: Error'
    return:
      type:
      - Error
  package: azure-iot-common
- uid: azure-iot-common.TwinDetachedError.constructor
  name: TwinDetachedError
  children: []
  type: constructor
  langs:
  - typeScript
  summary: ''
  syntax:
    content: 'new TwinDetachedError(message?: string)'
    parameters:
    - id: message
      type:
      - string
      description: ''
      optional: true
  package: azure-iot-common
